tic-tac-toe/src/tic_tac_toe/players.clj

```diff
@@ -0,0 +1,26 @@
+(ns tic-tac-toe.players
+  (:require [tic-tac-toe.marks :refer :all]
+            [tic-tac-toe.human-player :as human-player]
+            [tic-tac-toe.random-player :as random-player]
+            [tic-tac-toe.player-options :as player-options]))
+
+
+(def human-human
+  { X human-player/choose-move
+    O human-player/choose-move })
+
+(def random-human
+  { X random-player/choose-move
+    O human-player/choose-move })
+
+(def human-random
+  { X human-player/choose-move
+    O random-player/choose-move })
+
+(defn configure-players [game-option]
+(cond
+    (= game-option player-options/human-human-id) human-human
+    (= game-option player-options/random-human-id) random-human
+    (= game-option player-options/human-random-id) human-random
+    )
```

Comment:
I blogged about testing this module [here], as at first I was finding it difficult, and found an alternative. Then Danny suggested a slight simplification to the map declarations which made it easy to test: (http://gemcfadyen.github.io/georginam.com/apprenticeship/2016/03/17/apprenticeship-day-111.html)